CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+### Changed
```

Comment:
duplicate changelog?

Also maybe add a line saying that all user-facing features can be imported now directly from `tidy3d.plugins.autograd`?